repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -191,8 +192,62 @@ export const getLinesList = (
     return linesList;
 };
 
+export const getExemptedLinesList = (
+    userPeriodTicket: FlatFareGeoZoneTicket | PeriodGeoZoneTicket,
+    website: string,
+): Line[] => {
+    const linesList: Line[] = [];
+
+    const duplicateLines = userPeriodTicket.exemptedServices
+        ? userPeriodTicket.exemptedServices.map(service => ({
+              version: '1.0',
+              id: service.lineId,
+              Name: { $t: `Line ${service.lineName}` },
+              Description: { $t: service.serviceDescription },
+              Url: { $t: website },
+              PublicCode: { $t: service.lineName },
+              PrivateCode: service.lineId
```

Comment:
Can we change this to just 
{
                        type: 'txc:Line@id',
                        $t: service.lineId,
}

Instead of checking to see if the lineId is present? We use the lineId all over in this function, and this is the only place we check its there.